tests/Integration/ApiPlatform/ModuleEndpointTest.php

```diff
@@ -301,6 +311,43 @@ public function restResetModuleNotActive(array $module): void
         self::assertResponseStatusCodeSame(400);
     }
 
+    public function testUninstallModule()
+    {
+        $module = [
+            'technicalName' => 'bankwire',
+            'deleteFile' => false
+        ];
```

Comment:
No need for this variable, you can hard-code the value in your request parameter